Messenger/FBMApplication.m

```diff
@@ -0,0 +1,18 @@
+#import "FBMApplication.h"
+#import "AppDelegate.h"
+
+@implementation FBMApplication
+
+@dynamic delegate;
```

Comment:
It is not needed at all. :) Just some leftovers from my code experiments. I'll amend this commit.